lib/improver/tests/ensemble_calibration/ensemble_calibration/helper_functions.py

```diff
@@ -112,7 +113,7 @@ def set_up_probability_above_threshold_spot_cube(
     time_origin = "hours since 1970-01-01 00:00:00"
     calendar = "gregorian"
     tunit = Unit(time_origin, calendar)
-    cube.add_dim_coord(DimCoord([412227],
+    cube.add_dim_coord(DimCoord(np.array([412227], dtype=np.float32),
```

Comment:
I think time values should be float64, otherwise we cannot resolve 1 minute.

Reply:
Done.